docker/fc34/support-smatch.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # ---
-# git_url: git://repo.or.cz/smatch.git
-# git_commit: 9bb66fa2d7c73b3338a27fd6b38d7d509b2a1c1b
+# git_url: /global/home/users/artemp/scratch/.cache/mellanox/mkt/smatch.git
+# git_commit: 72c21a144a812cadbe349801da1b24bc331af256
```

Comment:
For some reason, the site where we were building this can't access the original URL.
This is specific to that ite and shouldn't be considered, probably.
Especially given that "mkt images" is not a must.

Reply:
IIRC if you preload the normal cache directory it doesn't require network access so long as the commit_id is already present. So these weird disconnected cases are solved by transfering the cache directory from some network connected machine

Reply:
What is normal cache directory?

Reply:
➜  kernel git:(master) ls ~/.cache/mellanox/mkt 
iproute2-next.git  rdma-core.git  simx.git  smatch.git  sparse.git  tc-build.git

Reply:
That’s the issue, it fails to download there from got saying connection refused.
Again, I don’t think we should consider it in MKT. this is obviously the issue on that site.

we just haven’t cleaned the version we ended up with for the sake of time consumption.